Towards A Social Platform.md

```diff
@@ -0,0 +1,88 @@
+# Long term goals for Social Platform
+
+Ultimately we are building towards incorporation of a social platform into KBase. 
+
+The goals of the social platform are to:
+
+1. Provide the means for users to organize and share their data and results in the system. 
+2. Provide a means for individual labs to organize their data and analyses in a way that is persistently available to the lab lead. 
```

Comment:
It's not separable-- but at its heart. We want people to feel they are engaged with their group communities (laboratory and projects) and the larger scientific community. The social interface has be constantly promoting sharing, interaction, and integration.